src/page/ChecksPage.tsx

```diff
@@ -290,14 +290,14 @@ export class ChecksPage extends PureComponent<Props, State> {
         </div>
         <section className="card-section card-list-layout-list">
           <ol className="card-list">
-            {filteredChecks.map(check => {
+            {filteredChecks.map((check, index) => {
               const checkId: number = check.id || 0;
               if (!check.id) {
                 return;
               }
```

Comment:
I know you didn't write this @rdubrock , still wanted to note that that should probably be refactored next time around. :) 
Because first, a default is used if `check.id` is falsy but then if `check.id` is falsy, we return early? => Invert the order and get rid of the default.